### ISSUE TITLE
nan of loss during training

### ISSUE THREAD

**Issue by yiColddrink** [thstkdgus35/EDSR-PyTorch#21]
Hello, when i  use my own training data with 11500 pictures, batch size =12 and scale = 2, a problem occurred
[Epoch 1]	Learning rate: 1.00e-4
[1200/10500]	[L1: 11.2929] 	21.0+60.3s
[2400/10500]	[L1: 8.8598] 	20.0+63.7s
[3600/10500]	[L1: 7.7807] 	20.0+69.5s
[4800/10500]	[L1: 7.1179] 	20.0+74.0s
[6000/10500]	[L1: 6.6776] 	20.1+69.4s
[7200/10500]	[L1: 6.3562] 	20.1+75.5s
[8400/10500]	[L1: 6.0822] 	20.0+75.2s
[9600/10500]	[L1: 5.8959] 	19.9+75.0s

Evaluation:
[Sequence x2]	PSNR: 37.031 (Best: 37.031 from epoch 1)
Time: 415.90s

[Epoch 2]	Learning rate: 1.00e-4
[1200/10500]	[L1: nan] 	20.2+46.7s
[2400/10500]	[L1: nan] 	20.1+48.3s
[3600/10500]	[L1: nan] 	20.1+54.2s
[4800/10500]	[L1: nan] 	20.1+60.7s
[6000/10500]	[L1: nan] 	20.2+62.0s
[7200/10500]	[L1: nan] 	20.1+63.0s
[8400/10500]	[L1: nan] 	20.1+68.9s
[9600/10500]	[L1: nan] 	20.1+68.6s

Evaluation:
[Sequence x2]	PSNR: 37.779 (Best: 37.779 from epoch 2)
Time: 403.14s
...
...
...
[Epoch 43]	Learning rate: 1.00e-4
[1200/10500]	[L1: nan] 	19.2+43.1s
[2400/10500]	[L1: nan] 	19.1+42.1s
[3600/10500]	[L1: nan] 	19.1+41.9s
[4800/10500]	[L1: nan] 	19.1+41.7s
[6000/10500]	[L1: nan] 	19.0+45.8s
[7200/10500]	[L1: nan] 	19.1+48.2s
[8400/10500]	[L1: nan] 	19.3+73.3s
[9600/10500]	[L1: nan] 	19.1+62.8s

Evaluation:
[Sequence x2]	PSNR: 39.445 (Best: 39.445 from epoch 43)
Time: 396.57s

**Comment by thstkdgus35** [comment]
Hello.

Although you got 'nan' error, PSNR increases as your training proceeds.

I cannot find exact answer, but I recommend you to print the loss value more frequently, to find when 'nan' starts to appear.

You can use ``--print_every`` argument for this task.

Thank you.
